server/helpers/sms.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { sanitizePhoneNumber, isValidNigerianPhoneNumber } from './funtions';
```

Comment:
Can we rename the `functions.js` file. It is a little confusing...

```suggestion
import { sanitizePhoneNumber, isValidNigerianPhoneNumber } from './functions';
```